beacon-chain/db/slasherkv/pruning_test.go

```diff
@@ -0,0 +1,286 @@
+package slasherkv
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	types "github.com/prysmaticlabs/eth2-types"
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	slashertypes "github.com/prysmaticlabs/prysm/beacon-chain/slasher/types"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/testutil/require"
+	"github.com/sirupsen/logrus"
+	logTest "github.com/sirupsen/logrus/hooks/test"
+	bolt "go.etcd.io/bbolt"
+)
+
+func TestMain(m *testing.M) {
+	logrus.SetLevel(logrus.DebugLevel)
+	logrus.SetOutput(ioutil.Discard)
+	m.Run()
+}
```

Comment:
Generally, we put `TestMain` into a separate file, in form `packagename/packagename_test.go`, otherwise it is not immediately clear if package has `TestMain` and in which file (with our current approach, it is immediately clear where to look for it, if one wants to disable log discarding for development for ex).